src/components/modal/v2/parts/views/NI/Content.jsx

```diff
@@ -0,0 +1,71 @@
+/** @jsx h */
+import { h, Fragment } from 'preact';
+import { useState, useRef } from 'preact/hooks';
+import Button from '../../Button';
+import Icon from '../../Icon';
+import ProductListLink from '../../ProductListLink';
+import Instructions from '../../Instructions';
+import styles from './styles/index.scss';
+import { useApplyNow } from '../../../lib';
+
+export const NI = ({ instructions, terms, buttonText, disclaimer, footer, listLink, contentBodyRef }) => {
+    const buttonRef = useRef();
+    const handleApplyNowClick = useApplyNow('Apply Now');
+    const [expandedState] = useState(false);
+
+    return (
+        <Fragment>
+            <style>{styles._getCss()}</style>
+            <div className="content__container">
+                <main className="main">
+                    <div className="content__body" ref={contentBodyRef}>
+                        <div className="content__row dynamic">
+                            <div className="content__col">
+                                <Instructions instructions={instructions} expandedState={expandedState} />
```

Comment:
```suggestion
                                <Instructions instructions={instructions} />
```
Since NI doesn't make use of ```expandedState``` we can remove this from here.